components/authz-service/server/v2/rules_test.go

```diff
@@ -74,9 +74,9 @@ func TestCreateRule(t *testing.T) {
 			grpctest.AssertCode(t, codes.InvalidArgument, err)
 			assert.Nil(t, resp)
 		}},
-		{"if the passed rule project id is invalid, returns 'invalid argument'", func(t *testing.T) {
+		{"if the passed rule project id is empty spaces, returns 'invalid argument'", func(t *testing.T) {
```

Comment:
Consistency with previous 2 files suggest "is whitespace":
```suggestion
		{"if the passed rule project id is whitespace, returns 'invalid argument'", func(t *testing.T) {
```